python/tvm/contrib/cutlass/gen_tensor_op.py

```diff
@@ -745,8 +745,8 @@ def get_batch_on_arg(arg_name, arg_shape):
             attrs["qkv"] = func_args[0]
             attrs["num_queries"] = s = annotations["num_queries"]
             attrs["num_keys"] = annotations["num_keys"]
-            if len(func_args) > 5 and not is_var_len:  # +1 for workspace, the last arg
-                attrs["bias"] = func_args[4]
+            if len(func_args) > 2 and not is_var_len:  # +1 for workspace, the last arg
```

Comment:
I guess cutlass side is mostly the interface change? 
Or is there any functional change?

Reply:
FuseOps is updated to inline the constant shape parameters. So there are fewer func_args here